snapcraft/plugins/python2.py

```diff
@@ -169,3 +165,10 @@ def snap_fileset(self):
         fileset.append('-usr/lib/python*/*/*/*/*/*/*/*/*/__pycache__/*.pyc')
         fileset.append('-usr/lib/python*/*/*/*/*/*/*/*/*/*/__pycache__/*.pyc')
         return fileset
+
+
+def _get_python2_include(root):
+    try:
+        return glob.glob(os.path.join(root, 'usr', 'include', 'python2*'))[0]
+    except IndexError:
+        raise EnvironmentError('python development headers not installed')
```

Comment:
This is a programmatic error though as python-dev should be a required package for the python plugin (as it is now).